src/WebdriverClassicDriver.php

```diff
@@ -155,7 +157,12 @@ public function reset(): void
 
     public function visit(string $url): void
     {
-        $this->getWebDriver()->navigate()->to($url);
+        try {
+            $this->getWebDriver()->navigate()->to($url);
+        } catch (TimeoutException|ScriptTimeoutException $e) {
+            // The W3C compatible Selenium Server.
```

Comment:
@aik099 sorry, very small final point: that comment is now a bit irrelevant.

Reply:
Done.